src/geocoder.js

```diff
@@ -28,14 +37,18 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
```

Comment:
According to [this](https://developers.google.com/maps/documentation/static-maps/get-api-key) document, premium plan users can use either an api key or the privateKey / clientId authentication method, the choice being up to the user.
According to [this](https://developers.google.com/maps/documentation/geocoding/usage-limits) document, even standard plan users get 50 requests per seconds, shared between server- and client side. As far as I understand, we should even be able to increase the requests per seconds when authenticated... Have a look at those documents, maybe I am getting something wrong.